### PR TITLE
Headless guide: Lighthouse -> chrome-launcher

### PR DESCRIPTION
**Pull request by paulirish**
With launcher shipped solo, we can clean up some more of this.

cc @samccone

### CODE REVIEW

File: src/content/en/updates/2017/04/headless-chrome.md [hunk 1]
```diff
@@ -164,35 +164,33 @@ platforms. Just look at that hard-coded path to Chrome :(
...
 
-    yarn add lighthouse
+    yarn add chrome-launcher
 
-**Example** - using `ChromeLauncher` to launch Headless
+**Example** - using `chrome-launcher` to launch Headless
 
 ```javascript
-const chromeLauncher = require('lighthouse/chrome-launcher/chrome-launcher');
+const chromeLauncher = require('chrome-launcher');
```
Comment (ebidel) [inline src/content/en/updates/2017/04/headless-chrome.md]: add:
```
const log = require('lighthouse-logger');

// Set logging for the launcher.
const LOG_LEVEL = 'info';
log.setLevel(LOG_LEVEL);
```

File: src/content/en/updates/2017/04/headless-chrome.md [hunk 2]
```diff
@@ -164,35 +164,33 @@ platforms. Just look at that hard-coded path to Chrome :(
...
  * Launches a debugging instance of Chrome.
  * @param {boolean=} headless True (default) launches Chrome in headless mode.
  *     False launches a full version of Chrome.
  * @return {Promise<ChromeLauncher>}
  */
-async function launchChrome(headless = true) {
-  return await chromeLauncher.launch({
+function launchChrome(headless=true) {
+  return chromeLauncher.launch({
     // port: 9222, // Uncomment to force a specific port of your choice.
```
Comment (ebidel) [inline src/content/en/updates/2017/04/headless-chrome.md]: and:
```
logLevel: LOG_LEVEL, // set logging for lighthouse.
```

**Comment by ebidel** [comment]
Bot has some warnings too.

File: src/content/en/updates/2017/04/headless-chrome.md (hunk 1 above)
Comment (paulirish) [inline src/content/en/updates/2017/04/headless-chrome.md]: technically this is optional, right?  can we put this into the launcher's readme instead?

File: src/content/en/updates/2017/04/headless-chrome.md (hunk 2 above)
Comment (paulirish) [inline src/content/en/updates/2017/04/headless-chrome.md]: hmmm.. AFAIK there is no `logLevel` option for chromeLauncher.. ?

File: src/content/en/updates/2017/04/headless-chrome.md (hunk 1 above)
Comment (ebidel) [inline src/content/en/updates/2017/04/headless-chrome.md]: It is, but I'd like to add it here so people don't have to go reading elsewhere. You could add "// Optional: set logging for launcher".

File: src/content/en/updates/2017/04/headless-chrome.md (hunk 2 above)
Comment (ebidel) [inline src/content/en/updates/2017/04/headless-chrome.md]: Ignore me. I'm mixing snippets. This example doesn't even use LH :)

File: src/content/en/updates/2017/04/headless-chrome.md (hunk 1 above)
Comment (ebidel) [inline src/content/en/updates/2017/04/headless-chrome.md]: log.setLevel('info');

File: src/content/en/updates/2017/04/headless-chrome.md (hunk 2 above)
Comment (ebidel) [inline src/content/en/updates/2017/04/headless-chrome.md]: done

**Comment by ebidel** [comment]
@petele need your merging powers :)

**Pull request merged**